package.json

```diff
@@ -35,9 +35,9 @@
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
     "@babel/preset-typescript": "7.3.3",
-    "@fortawesome/fontawesome-svg-core": "1.2.36",
-    "@fortawesome/free-solid-svg-icons": "5.15.4",
-    "@fortawesome/react-fontawesome": "0.1.17",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
```

Comment:
Could we lock the version? `^6.1.1` --> `6.1.1`

It would help making sure that the dependencies' versions are predictable, otherwise sometimes it'll be `6.1.1` here and maybe `6.2.x` in `yarn.lock`.